### PR TITLE
[WIP] Test if not using Travis Python fixes timeouts

### PR DESCRIPTION
**Pull request by ezyang**
Signed-off-by: Edward Z. Yang <ezyang@fb.com>

### CODE REVIEW

**Comment by ezyang** [comment]
Wow... the runtime failure on deadsnakes is... exciting to say the least.

```
Oct 29 02:50:43 + python test/run_test.py --include nn --verbose
*** Error in `python': munmap_chunk(): invalid pointer: 0x00007f8fbc489b50 ***
Oct 29 02:50:43 .jenkins/pytorch/test.sh: line 88: 17115 Aborted                 python test/run_test.py --include nn --verbose
```

**Comment by ezyang** [comment]
Tabling this for now, figuring out why there's a memory with deadsnakes python is going to take some serious debugging.

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # IMPORTANT: To update Docker image version, please search and update ":{previous_version}"
```
Comment (anderspapitto) [inline .circleci/config.yml]: either you're wrong or this comment is?

Comment (anderspapitto) [reply .circleci/config.yml]: i.e. I don't see any modifications of a `previous_version` in this diff, and the comment seems to say that you need one

Comment (ezyang) [reply .circleci/config.yml]: Err, the change is right below. previous_version here is 253.

Comment (anderspapitto) [reply .circleci/config.yml]: it says 'update previous_version and ALSO the version number below' - are they not two separate things?

Comment (ezyang) [reply .circleci/config.yml]: Version number below is https://github.com/pytorch/pytorch/pull/13231/files#diff-1d37e48f9ceff6d8030570cd36286a61R3

Update previous version is all occurrences of the number in the yaml: e.g., https://github.com/pytorch/pytorch/pull/13231/files#diff-1d37e48f9ceff6d8030570cd36286a61R429

**Pull request closed without merging**